webapi/SemanticKernelExtensions.cs

```diff
@@ -140,6 +141,29 @@ private static void AddSemanticTextMemory(this IServiceCollection services)
                     return new AzureCognitiveSearchMemoryStore(config.AzureCognitiveSearch.Endpoint, config.AzureCognitiveSearch.Key);
                 });
                 break;
+            case MemoriesStoreOptions.MemoriesStoreType.Chroma:
+                if (config.Chroma == null)
+                {
+                    throw new InvalidOperationException("MemoriesStore type is Chroma and Chroma configuration is null.");
+                }
+
+                services.AddSingleton<IMemoryStore>(sp =>
+                {
+                    HttpClient httpClient = new(new HttpClientHandler { CheckCertificateRevocationList = true });
+                    var endPointBuilder = new UriBuilder(config.Chroma.Host);
+                    endPointBuilder.Port = config.Chroma.Port;
+
+                    return new ChromaMemoryStore(
+                        httpClient: httpClient,
+                        endpoint: endPointBuilder.ToString(),
+                        logger: sp.GetRequiredService<ILogger<IChromaClient>>()
+                    );
+                });
+                services.AddScoped<ISemanticTextMemory>(sp => new SemanticTextMemory(
```

Comment:
Remove lines 162 to 165. They have already been factored out of each individual case of the switch and put below the switch instead of being duplicated all over.